google-cloud-clients/google-cloud-pubsub/pom.xml

```diff
@@ -78,6 +78,12 @@
       <artifactId>grpc-google-iam-v1</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
```

Comment:
Do we really need to add `opencensus-impl` here?

Reply:
We should not, only the final app should do this.

Reply:
These are the test dependencies rather than the library dependencies. I use the implementation to test OpenCensusUtil.